.github/workflows/flannel.yml

```diff
@@ -0,0 +1,30 @@
+name: flannel-images
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Since this is only for the flannal images we should restrict these to that folder. https://github.blog/changelog/2019-09-30-github-actions-event-filtering-updates/

Reply:
Ok